### PR TITLE
[ticket/16279] Add permission for Emoji in topic title

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-16279

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

https://tracker.phpbb.com/browse/PHPBB3-16279

### CODE REVIEW

File: phpBB/language/en/acp/permissions_phpbb.php [hunk 1]
```diff
@@ -79,6 +79,7 @@
 	'ACL_U_SAVEDRAFTS'	=> 'Can save drafts',
 	'ACL_U_CHGCENSORS'	=> 'Can disable word censors',
 	'ACL_U_SIG'			=> 'Can use signature',
+	'ACL_U_EMOJI'		=> 'Can use Emoji in topic title',
```
Comment (Crizz0) [inline phpBB/language/en/acp/permissions_phpbb.php]: Emoji => emojis

Comment (mrgoldy) [reply phpBB/language/en/acp/permissions_phpbb.php]: I would still stick with just `emoji`, as it's the plural form aswell.

Comment (Crizz0) [reply phpBB/language/en/acp/permissions_phpbb.php]: According to those example sentences: https://www.lexico.com/definition/emoji the "emojis" is for plural more common.

**Comment by vinny** [comment]
Add permission for this is really necessary?

**Comment by DavidIQ** [comment]
The internet is more juvenile than we thought when the functionality was added in so there were some concerns that bubbled up.

**Comment by marc1706** [comment]
If you want to have this permission enabled by default, then your migration still needs to actually set this permission for users.

**Comment by marc1706** [comment]
Merged via https://github.com/phpbb/phpbb/commit/fcca5c5b582c43353be73ae50afb631e51407bbf

**Pull request closed without merging**